marklogic-data-hub/src/main/java/com/marklogic/hub/flow/impl/FlowRunnerImpl.java

```diff
@@ -118,8 +119,9 @@ public FlowRunnerImpl(String host, String username, String password) {
      */
     public FlowRunnerImpl(HubConfig hubConfig) {
         this.hubConfig = hubConfig;
-        this.stepRunnerFactory = new StepRunnerFactory(this.hubConfig);
-        this.stepRunnerFactory.setStepDefinitionProvider(new MarkLogicStepDefinitionProvider(this.hubConfig.newStagingClient(null)));
+        this.flowManager = new FlowManagerImpl(hubConfig);
+        this.stepRunnerFactory = new StepRunnerFactory(hubConfig);
+        this.stepRunnerFactory.setStepDefinitionProvider(new MarkLogicStepDefinitionProvider(hubConfig.newStagingClient(null)));
```

Comment:
You can get rid of this line of code now that StepRunnerFactory is doing this part itself in its constructor.